i18n/bg.json

```diff
@@ -13,7 +13,7 @@
   "about.hashee": "Хеш на EE компилация:",
   "about.hashwebapp": "Webapp билд хеш:",
   "about.licensed": "Лицензиран на:",
-  "about.notice": "Използването на Mattermost е възможно благодарение на софтуер с отворен код използван в нашите [сървърно](!https://about.mattermost.com/platform-notice-txt/), [настолно](!https://about.mattermost.com/desktop-notice-txt/) и [мобилно](!https://about.mattermost.com/mobile-notice-txt/) приложения.",
+  "about.notice": "Използването на Mattermost е възможно благодарение на софтуер с отворен код използван в нашите [сървърно](!https://github.com/mattermost/mattermost-server/blob/master/NOTICE.txt), [настолно](!https://github.com/mattermost/desktop/blob/master/NOTICE.txt) и [мобилно](!https://github.com/mattermost/mattermost-mobile/blob/master/NOTICE.txt) приложения.",
```

Comment:
Note that this might cause issues if these conflict with the strings that are automatically submitted from the translation server. I'm not sure how we've dealt with stuff like this in the past since we've never done this sort of mass update before

Reply:
I think it'd have a similar effect on the translators since they'd still have to update the strings in this release, but afterwards it would mean we could change the URLs freely. I do think it would be a good idea though since it removes any risk of a URL getting changed accidentally by a translator.

Depending on how far we want to go with that, there's a way in the i18n library that we use where we could make the translated string look like
```
"some.string": "this is <link>a link to some web site</link>"
```
and then the code would be entirely responsible for what the link is and what it looks like as opposed to now where the translator has to deal with a Markdown directly

Reply:
> `"some.string": "this is <link>a link to some web site</link>"`
+1

Reply:
Awesome! Thanks, @hmhealey! I'm capturing this as a future enhancement. 

In terms of the domain switchover we're facing in the short term, what would you recommend as next steps? Should I:
- leave my string updates as-is in this repo (and notify the translation community that a LOT of strings are going to look like they changed but have likely only changed in terms of the link URL)
- revert my string updates in all language files except ``en.json`` (and if we go this route, will those URL changes get picked up via a round-trip through Weblate, perhaps?)